python/hsfs/core/inbuilt_transformation_function.py

```diff
@@ -0,0 +1,153 @@
+
+    @staticmethod
+    def generate_min_max_scaler():
+        output_type = "double"
+        source_code = """
+            # Min-Max scaling
+            def min_max_scaler(value, min_value, max_value):
+                return (value - min_value) / (max_value - min_value)
+            """
+        return source_code, output_type
+
+    @staticmethod
+    def min_max_scaler_stats(content, feature_name):
+        min_value = None
+        max_value = None
+        for col in content["columns"]:
+            if col["column"] == feature_name:
+                if col["dataType"] not in ["Integral", "Fractional", "Decimal"]:
+                    raise ValueError("Can't compute min_max_scaler for this type")
+                if "minimum" in col:
+                    min_value = col["minimum"]
+                if "maximum" in col:
+                    max_value = col["maximum"]
+
+        if min_value is None and max_value is None:
+            raise FeatureStoreException(
+                "Feature {feature_name:} doesn't have minimum and/or maximum values computed. Thus can't use "
+                "min_max_scaler method".format(feature_name=feature_name)
+            )
+        return min_value, max_value
+
+    @staticmethod
+    def generate_standard_scaler():
+        output_type = "double"
+        source_code = """
+            # Standardization / zcore
+            def standard_scaler(value, mean, std_dev):
+                return (value - mean) / std_dev
+            """
+        return source_code, output_type
+
+    @staticmethod
+    def standard_scaler_stats(content, feature_name):
+        mean = None
+        std_dev = None
+        for col in content["columns"]:
+            if col["column"] == feature_name:
+                if col["dataType"] not in ["Integral", "Fractional", "Decimal"]:
+                    raise ValueError("Can't compute standard_scaler for this type")
+                if "mean" in col:
+                    mean = col["mean"]
+                if "stdDev" in col:
+                    std_dev = col["stdDev"]
+
+        if mean is None and std_dev is None:
+            raise FeatureStoreException(
+                "Feature {feature_name:} doesn't have mean and/or standard deviation computed. Thus can't use "
+                "standard_scaler method".format(feature_name=feature_name)
+            )
+        return mean, std_dev
+
+    @staticmethod
+    def generate_robust_scaler():
+        output_type = "double"
+        source_code = """
+                # Robust scaling
+                def robust_scaler(value, p25, p50, p75):
+                    return (value - p50) / (p75 - p25)
+                """
+        return source_code, output_type
+
+    @staticmethod
+    def robust_scaler_stats(content, feature_name):
+        percentiles = None
+        for col in content["columns"]:
+            if col["column"] == feature_name:
+                if col["dataType"] not in ["Integral", "Fractional"]:
```

Comment:
"Decimal" can be added to the list. Since https://github.com/logicalclocks/deequ/pull/10 they are supported as well.